trunk/workflow-manager/src/test/java/org/mitre/mpf/wfm/camelOps/TestDetectionSplitter.java

```diff
@@ -197,7 +197,7 @@ public void testMediaSpecificPropertiesOverrideWithExif() throws Exception {
         testExifWithSpecificProperty(MpfConstants.SEARCH_REGION_BOTTOM_RIGHT_Y_DETECTION_PROPERTY, "-1",true);
```

Comment:
Let's rename this test to `testMediaSpecificPropertiesOverrideWithFrameTransforms`. "WITH EXIF" is a bit misleading since it only applies to flip and rotation.